pkg/ui/src/views/statements/statementsTable.tsx

```diff
@@ -129,6 +137,12 @@ function makeCommonColumns(statements: AggregateStatistics[])
   const latencyBar = latencyBarChart(statements);
 
   return [
+    {
```

Comment:
I'd say while we're here, change the default sort column to latency (it's what people care about…)